### DISCUSSION TITLE
Multiple videos section with 2 or more list in a single page

### DISCUSSION THREAD

**Discussion by shejinalt** [videojs/video.js#7839] [General]
Hi,

I have multiple video sections  and each section have it's own video list. when I click item on list its correspond video frame is not loading. pleas find this fiddle demo

https://jsfiddle.net/shejinalt/c28qLvg4/1/

**Comment by mister-ben** [comment]
There are multiple issues in your example.

* Using http URLs on an https page.
* Trying to use the same `id` on two players. They must be unique. Your click handler needs to know which player it wants to work with.
* `<source>` elements with junk URLs. Just leave out the `<source>` if you want to init ht eplayer without a source.
* Don't call `player.load()`. The player calls load on the video el when needed, calling it an extra time can cause problems.

Hope that helps.

**Reply by shejinalt** [reply]
you are right; the id should be unique. Is it possible to init with className.
 I can't find className init in the docs.
